test/testutil/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright Soramitsu Co., Ltd. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-add_library(logger_for_tests INTERFACE)
+add_library(logger_for_tests INTERFACE ../core/runtime/small_lru_cache_test.cpp)
```

Comment:
Why?

Reply:
Because CLion likes to deliver surprises.